workshops/04-Retrospective-Planning-Workshop/README.md

```diff
@@ -0,0 +1,75 @@
+# Retrospective Planning
+
+## Exercises
+
+1. Before continuing you should read the information below.
+2. As a whole group, brainstorm the purpose of a retrospective for your team.
+3. As a whole group, brainstorm what you think an ideal retrospective facilitator;
+    - Should do
+    - Should not do
+4. Individually think of some specific challenges that would be hard to uncover or work past in a retrospective.
+5. In pairs, work together to design or choose two retrospective plans
+    - Write down what extra materials you will need to run them
+    - (If applicable) Decide between you who will facilitate which plan
+6. Present back to the group your retrospective plans and any specific reasons for choosing them given what you discovered in 2, 3 and 4.
+
+## The Prime Directive
+
+Retrospectives are meant to be a healthy,
+fun activity in which the team discusses the way they work together.
+
+This is in direct contrast to blame culture,
+where people feel the need to hide what they are struggling with. 
+
+A healthy "safe" agile culture is one where every team member feels comfortable
+enough to discuss their own and other's mistakes.
+
+We remember this quote every retrospective as a reminder of this:
+
+> "Regardless of what we discover, 
+> we understand and truly believe that everyone did the best job they could, 
+> given what they knew at the time, 
+> their skills and abilities, 
+> the resources available, 
+> and the situation at hand." 
+>
+> Norm Kerth in _Project Retrospectives: A Handbook for Team Review_
+
+## A basic retrospective
+
+* What went well?
+* What did not go well?
+* What can be done differently?
+* What do we still not understand?
+
+## Facilitating retrospectives
+
+Extending from the basic questions, a facilitator may choose to get creative 
+with formats. To make the retrospective _fun_.
```

Comment:
"The author does not approve of _fun_" :p

Might want an exclamation mark here!